terragrunt/aws/api/inputs.tf

```
@@ -0,0 +1,31 @@
+variable "api_auth_token" {
+  description = "The API token needed for all requests to the API."
+  type        = string
+  sensitive   = true
+}
+
+variable "url_shortener_table_name" {
+  description = "DynamoDB table name for the URL shortener"
+  type        = string
+}
+
+variable "ecr_repository_arn" {
+  description = "Arn of the ECR Repository"
+  type        = string
+}
+
+variable "ecr_repository_url" {
+  description = "URL of the URL shortener ECR"
+  type        = string
+}
+
+variable "api_id" {
+  description = "Api security group Id"
+  type        = string
+}
+
+variable "private_subnet_ids" {
+  description = "List of the URL Shortner VPC private subnet ids"
+  type        = string
```

Since this is a list of strings (I think 😅) we can set the type like so:
```suggestion
  type        = list(string)
```

---

Thank you for the suggestions - it is done!